product_list.php

```diff
@@ -3,34 +3,23 @@
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
 require_once 'config/constants.php';
+require_once 'libraries/db.php';
```

Comment:
require_once 'config/constants.php';

No need to include the file again. As you already including this file in db.php